doc/source/f2py/allocarr_session.dat

```diff
@@ -1,8 +1,9 @@
 >>> import allocarr 
->>> print allocarr.mod.__doc__
-b - 'f'-array(-1,-1), not allocated
-foo - Function signature:
-  foo()
+>>> print(allocarr.mod.__doc__)
+'f'-
```

Comment:
Need to restore the names of module data items.

Reply:
I'm restoring the names but this is not the current output from f2py - depends on #15325